src/main.cpp

```diff
@@ -90,6 +90,8 @@ AGEHook<0x62B068>::Type<int> timeOfDay;
 
 AGEHook<0x627518>::Type<int> carLights;
 
+AGEHook<0x6B2E41>::Type<int> gameState;   //0 = menu, 1 = level loaded
```

Comment:
Actually, this was already defined in 'mm2_common.h', just under a different name ('splashScreen').

You could either do `if(splashScreen == 1)` or rename 'splashScreen' to 'gameState'. Up to you.